templates/Visua11ySettings.jsx

```diff
@@ -148,21 +148,21 @@ export default function Visua11ySettings(props) {
             <div className="visua11ysettings__item-option-container" role='radiogroup' aria-labelledby='fontsize'>
 
               <div className='visua11ysettings__item-option'>
-                <input type='radio' value={config._fontSize._small} checked={visua11y.fontSize === config._fontSize._small} id='fontSize-small' name='fontSize' aria-label='Font Size - Small' onKeyPress={onKeyPress} onChange={onChange} onFocus={onItemFocus} onBlur={onItemBlur} />
+                <input type='radio' value={config._fontSize._small} checked={visua11y.fontSize === config._fontSize._small} id='fontSize-small' name='fontSize' aria-label={config._fontSize.smallLabel} onKeyPress={onKeyPress} onChange={onChange} onFocus={onItemFocus} onBlur={onItemBlur} />
```

Comment:
This will make labels a numerical value and for medium font size, plainly say 'medium'. Is that a sufficient label?

Reply:
https://github.com/cgkineo/adapt-visua11y/blob/c8819e33491e86e95e094e4a723ea4185901e359/example.json#L65-L75

> this will make labels a numerical value

`smallLabel` should read `"Small"`

The labels should not be hardcoded in the jsx as they will need to be translated and therefore must be updatable from the json.

Having it just read the option labels `"Small"`, `"Medium"` etc is perfect. The group title will be read on entry and exit. This is how the mcq currently works.